sequencing/ATACSeq/preprocessing/countMTReads.sh

```diff
@@ -1,22 +1,20 @@
-## get the number of reads aligned to MT chromosome from samtools idxstats output
-
-
-## EXECUTION
-# sh ATACSeq/countMTReads.sh 
-
-## REQUIRES the following variables in config file
-# ALIGNEDDIR
-
-## REQUIRES the following software
-# -
-
-## INPUT
-# -
-
-## OUTPUT
-# ${ALIGNEDDIR}/countMTReads.txt
-
-
+## ====================================================================================================##
```

Comment:
I know it's being called from 4_collateStage1QCMetrics.sh, but could you add a shebang line to the top of this script anyways?